charts/immich/values.yaml

```diff
@@ -8,207 +8,119 @@
+      main: {}
+        # resources:
+        #   limits:
+        #     nvidia.com/gpu.shared: '1'
 
-podAnnotations:
-  checksum/config: "{{ .Values.immich.configuration | toYaml | sha256sum }}"
+# see https://bjw-s-labs.github.io/helm-charts/docs/common-library/storage/types/persistentVolumeClaim/#existing-claim
+persistence:
+   library:
+     accessMode: ReadWriteOnce
+     size: 100Gi
+     nameOverride: ''
+     storageClass: ''
+   # Optional: Set this to pvc to avoid downloading the ML models every start.
+   # machine-learning-cache:
+   #   enabled: true
+   #   accessMode: ReadWriteOnce
+   #   size: 10Gi
+   # external-1:
+   #   type: persistentVolumeClaim
+   #   existingClaim: ''
+   #   advancedMounts:
+   #     server:
+   #       main:
+   #         - path: /usr/src/app/external-1
 
-serviceMonitor:
-  server:
-    serviceName: server
-    endpoints:
-      - port: metrics-api
-        scheme: http
-      - port: metrics-ms
-        scheme: http
+# ingress:
+#   main:
+#     annotations: {}
+#       # proxy-body-size is set to 0 to remove the body limit on file uploads
+#       # nginx.ingress.kubernetes.io/proxy-body-size: "0"
+#     hosts:
+#       - host: immich.local
+#         paths:
+#           - path: "/"
+#             service:
+#               identifier: server
+#     tls: []
 
-# persistence:
-#   server:
-#     identifier: server
-#     advancedMounts:
-#       - type: configMap
-#         name: "{{ .Release.Name }}-immich-config"
-#       - name: immich-library
-#         path: /usr/src/app/upload
-#         accessMode: ReadWriteMany
-#         size: 1Ti
-#         storageClass: # provide your storage class
-#         # existingClaim: immich-library Provide your existing PVC
-#       - path: /usr/src/app/external
-#         name: immich-external
-#         accessMode: ReadWriteMany
-#         size: 1Ti
-#         storageClass: # provide your storage class
-#         # existingClaim: immich-external Provide your existing PVC
-#   machine-learning:
-#     identifier: machine-learning
-#     advancedMounts:
-#       - name: cache
-#         mountPath: /cache
-#         size: 10Gi
-#         # Optional: Set this to pvc to avoid downloading the ML models every start.
-#         type: emptyDir
-#         accessMode: ReadWriteMany
-#         storageClass: # provide your storage class
+rawResources:
+  psql:
+    enabled: false
+    apiVersion: postgresql.cnpg.io/v1
+    kind: Cluster
+    spec:
+      spec:
+        # storage:
+        #   size: 8Gi
+        #   storageClass: ''
+        bootstrap:
+          initdb:
+            secret:
+              name: immich-psql
 
-ingress:
-  server:
+  redis:
     enabled: false
-    annotations:
-      # proxy-body-size is set to 0 to remove the body limit on file uploads
-      nginx.ingress.kubernetes.io/proxy-body-size: "0"
-    hosts:
-      - host: immich.local
-        paths:
-          - path: "/"
-    tls: []
+    apiVersion: redis.redis.opstreelabs.in/v1beta2
```

Comment:
similar thing here, we could ensure not both redis variants can be active at the same time?